src/Plugin/DataType/StrawberryValuesViaJmesPathFromJson.php

```diff
@@ -85,6 +85,11 @@ public function process($langcode = NULL)
       }
       // This is an array, don't double nest to make the normalizer happy.
       $jmespath_result_to_expose = array_filter($jmespath_result_to_expose);
+      foreach($jmespath_result_to_expose as $i => $v) {
```

Comment:
Wait. What is this?

Reply:
that's "also includes fix for cases when the jmespath_result is not a string" - unrelated to this issue. Reject if you prefer.

Reply:
No, not rejecting but trying to figure out how this would work? So you mean if not a string, you want to actually index the JSON string?

Reply:
Also, if new please add a comment on top (before the foreach) explaining the expected behavior.

This scares me a bit because if you try to index let's say as:image... it would for each image the full JSON as a multivalued field? Huge string?

Reply:
Let's roll with this, I should really fix StrawberryfieldJsonHelper::arrayIsMultiSimple instead (may in a next commit or we can roll with your fix until 1.1.0